vignettes/testdat.Rmd

```diff
@@ -37,7 +37,7 @@ Features include:
 As an extension of testthat, testdat uses the same basic testing framework.
```

Comment:
Need to review the links in this vignette. I reorganised the functions and broke some things, e.g. ?value-expectations.

See https://github.com/socialresearchcentre/testdat/issues/34

Reply:
Ah yep - I'll mark this resolved for the mo and come back to it on that issue - it's against the CRAN milestone